plugin/types/types.pb.go

```diff
@@ -42,41 +98,17 @@ var EncodingType_value = map[string]int32{
 func (x EncodingType) String() string {
 	return proto.EnumName(EncodingType_name, int32(x))
 }
-func (EncodingType) EnumDescriptor() ([]byte, []int) {
-	return fileDescriptor_types_89e1090194eb9db5, []int{0}
-}
+func (EncodingType) EnumDescriptor() ([]byte, []int) { return fileDescriptorTypes, []int{0} }
 
 type PluginCode struct {
-	Name                 string   `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
-	Input                []byte   `protobuf:"bytes,2,opt,name=input,proto3" json:"input,omitempty"`
-	XXX_NoUnkeyedLiteral struct{} `json:"-"`
-	XXX_unrecognized     []byte   `json:"-"`
-	XXX_sizecache        int32    `json:"-"`
+	Name  string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
+	Input []byte `protobuf:"bytes,2,opt,name=input,proto3" json:"input,omitempty"`
 }
 
-func (m *PluginCode) Reset()         { *m = PluginCode{} }
-func (m *PluginCode) String() string { return proto.CompactTextString(m) }
-func (*PluginCode) ProtoMessage()    {}
-func (*PluginCode) Descriptor() ([]byte, []int) {
-	return fileDescriptor_types_89e1090194eb9db5, []int{0}
-}
-func (m *PluginCode) XXX_Unmarshal(b []byte) error {
-	return xxx_messageInfo_PluginCode.Unmarshal(m, b)
-}
-func (m *PluginCode) XXX_Marshal(b []byte, deterministic bool) ([]byte, error) {
-	return xxx_messageInfo_PluginCode.Marshal(b, m, deterministic)
-}
-func (dst *PluginCode) XXX_Merge(src proto.Message) {
-	xxx_messageInfo_PluginCode.Merge(dst, src)
-}
-func (m *PluginCode) XXX_Size() int {
-	return xxx_messageInfo_PluginCode.Size(m)
-}
-func (m *PluginCode) XXX_DiscardUnknown() {
-	xxx_messageInfo_PluginCode.DiscardUnknown(m)
-}
-
-var xxx_messageInfo_PluginCode proto.InternalMessageInfo
+func (m *PluginCode) Reset()                    { *m = PluginCode{} }
+func (m *PluginCode) String() string            { return proto.CompactTextString(m) }
+func (*PluginCode) ProtoMessage()               {}
+func (*PluginCode) Descriptor() ([]byte, []int) { return fileDescriptorTypes, []int{0} }
```

Comment:
Looks like this was generated by a different `protoc` version than usual. Rerun `make deps` in `loomchain` to make sure you have the correct version. The `XXX_` methods shouldn't be getting removed.

Reply:
yeah. it was wrong version. Put the XXX_ back.